design-system/atoms/chart-legend/adapters/default/default.adapter.tsx

```diff
@@ -23,7 +23,7 @@ export function ChartLegendDefaultAdapter<C extends ElementType = "div">({
 
       <Typo
         as={"div"}
-        size={size === "m" ? "xs" : "xxs"}
+        size={size === "m" ? "xs" : "sm"}
```

Comment:
Would it be worth mapping ?

Reply:
this condition only exists in one place but can output the condition higher!